src/components/editor/Editor.tsx

```diff
@@ -5,74 +5,62 @@ import {selectEditorState} from 'store/selectors';
 
 import {AnnotationPriorities, annotationPrioritiesLabels} from '../consts';
 import styles from './Editor.scss';
+import {DragTracker, IVec2} from '../../utils/move';
 import PriorityButton from './priority-button/PriorityButton';
 import {Modal, Popup} from 'semantic-ui-react';
-import {DraggableWidget} from 'components/widget';
+import Widget from '../widget/Widget';
+import {IEditorState, IEditorProps, IEditorForm} from './interfaces';
+import { DraggableWidget } from 'components/widget';
```

Comment:
`import Widget` and this one can be merged into one import - `import Widget, { DraggableWidget }` ... `